scripts/gdal/gdal_preset.py

```diff
@@ -36,6 +36,10 @@
     "-co",
     # Set Max Z Error to 1mm
     "max_z_error=0.001",
+    # Force all DEMS to AREA to be consistent
+    # input tiffs vary between AREA or POINT
+    "-mo",
+    "AREA_OR_POINT=AREA",
```

Comment:
Complete NIT but the convention seems to be `AREA_OR_POINT=Area` (or `AREA_OR_POINT=Point`).